01-Login/server.py

```diff
@@ -1,110 +1,105 @@
 """Python Flask WebApp Auth0 integration example
 """
-from functools import wraps
+
 import json
 from os import environ as env
-from werkzeug.exceptions import HTTPException
+from urllib.parse import quote_plus, urlencode
 
-from dotenv import load_dotenv, find_dotenv
-from flask import Flask
-from flask import jsonify
-from flask import redirect
-from flask import render_template
-from flask import session
-from flask import url_for
 from authlib.integrations.flask_client import OAuth
-from six.moves.urllib.parse import urlencode
-
-import constants
+from dotenv import find_dotenv, load_dotenv
+from flask import Flask, jsonify, redirect, render_template, session, url_for
+from werkzeug.exceptions import HTTPException
 
 ENV_FILE = find_dotenv()
 if ENV_FILE:
     load_dotenv(ENV_FILE)
 
-AUTH0_CALLBACK_URL = env.get(constants.AUTH0_CALLBACK_URL)
-AUTH0_CLIENT_ID = env.get(constants.AUTH0_CLIENT_ID)
-AUTH0_CLIENT_SECRET = env.get(constants.AUTH0_CLIENT_SECRET)
-AUTH0_DOMAIN = env.get(constants.AUTH0_DOMAIN)
-AUTH0_BASE_URL = 'https://' + AUTH0_DOMAIN
-AUTH0_AUDIENCE = env.get(constants.AUTH0_AUDIENCE)
-
-app = Flask(__name__, static_url_path='/public', static_folder='./public')
-app.secret_key = constants.SECRET_KEY
+app = Flask(__name__, static_url_path="/public", static_folder="./public")
+app.secret_key = env.get("APP_SECRET_KEY")
 app.debug = True
 
 
 @app.errorhandler(Exception)
 def handle_auth_error(ex):
     response = jsonify(message=str(ex))
-    response.status_code = (ex.code if isinstance(ex, HTTPException) else 500)
+    response.status_code = ex.code if isinstance(ex, HTTPException) else 500
     return response
 
 
+def fetch_token(name, request):
+    token = OAuth2Token.find(name=name, user=request.user)
```

Comment:
I know this is `authlib` boilerplate code but IDEs are complaining about this variable not defined.

Reply:
Thanks @ilayn, I've not found any guidance on how we might circumvent that, as it appears to be a global injected by the library and doesn't use an import keyboard. IDE warning aside, it does appear to work fine at least